core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -912,6 +914,7 @@ public function loadDetails()
 					if ($objParentPage->includeCache)
 					{
 						$this->cache = $this->cache !== false ? $this->cache : $objParentPage->cache;
+						$this->alwaysLoadFromCache = $this->alwaysLoadFromCache !== false ? $this->alwaysLoadFromCache : $objParentPage->alwaysLoadFromCache;
```

Comment:
Can probably be simplified to `$this->alwaysLoadFromCache = $this->alwaysLoadFromCache || $objParentPage->alwaysLoadFromCache;`

Reply:
I wanted to stick to the coding style of all the other properties.